runtime/compiler/env/j9methodServer.cpp

```diff
@@ -1060,13 +1054,15 @@ TR_ResolvedJ9JITServerMethod::fieldOrStaticName(I_32 cpIndex, int32_t & len, TR_
    J9ROMFieldRef * ref = (J9ROMFieldRef *) (&romCPBase()[cpIndex]);
    J9ROMNameAndSignature * nameAndSignature = J9ROMFIELDREF_NAMEANDSIGNATURE(ref);
 
-   if (inROMClass(nameAndSignature))
+   if (JITServerHelpers::isAddressInROMClass(nameAndSignature, romClassPtr()))
```

Comment:
Since `romClassPtr()` is repeated 4 times, mayber we should call it once and save the answer to a local variable (being a virtual call could be a bit expensive)